docs/guide/02-grammar-and-types.md

```diff
@@ -79,15 +79,15 @@ x = 5 // this will not raise an error
 
 ## Data Structures and Data types
 
-- There are 
-  - Boolean: `kweli` that represents true and `uwongo` that represents false
-  - null: `tupu` denotes a null value
-  - Number: Could be both Integers and floats. For example `42` or `3.14159`
-  - Strings: A sequence of characters that represents a text value. For example `Habari`
-  - Lists: `Orodha` an ordered collection of items. Can hold multiple data types: for example `[1,2,3]` or `[1, "orodha", kweli]`
-  - Regular Expressions (RegEx) : These are patterns used to match character combinations in strings. 
-  - Date : Used to represent a date object.
-  - Kamusi : Works like a dictionary by storing items as **key/value pairs**
+- There are 8 Data structures and Types in Swahili.
+  1. Boolean: `kweli` that represents true and `uwongo` that represents false
+  2. null: `tupu` denotes a null value
+  3. Number: Could be both Integers and floats. For example `42` or `3.14159`
+  4. Strings: A sequence of characters that represents a text value. For example `Habari`
+  5. Lists: `Orodha` an ordered collection of items. Can hold multiple data types: for example `[1,2,3]` or `[1, "orodha", kweli]`
+  6. Regular Expressions (RegEx) : These are patterns used to match character combinations in strings. 
+  7. Date : Used to represent a date object.
+  8. Kamusi : Works like a dictionary by storing items as **key/value pairs**
```

Comment:
Naming inconsistency (all the rest are in English)

Reply:
Do we call them Dictionaries or Objects?